rest_framework_ccbv/renderers.py

```diff
@@ -1,22 +1,25 @@
-
 import json
+from collections import defaultdict
 
-from inspector import Inspector
-from jinja_utils import templateEnv
-from config import REST_FRAMEWORK_VERSIONS, VERSION, BASE_URL
-from itertools import ifilter
+from .inspector import Inspector
+from .jinja_utils import templateEnv
+from .config import REST_FRAMEWORK_VERSIONS, VERSION, BASE_URL
 
 
 class BasePageRenderer(object):
 
     def __init__(self, klasses):
-        self.klasses = klasses
+        grouped_klasses = defaultdict(list)
+        for klass in klasses:
+            module = klass.__module__
+            grouped_klasses[module].append(klass)
+        self.grouped_klasses = grouped_klasses
```

Comment:
Better to convert to dict here, to avoid unexpected behavior: KeyError will never happen.

So please replace with `self.grouped_klasses = dict(grouped_klasses)`